timbr/DetailsTableViewController.m

```diff
@@ -24,15 +24,59 @@ - (void)viewDidLoad {
     [super viewDidLoad];
     [self.tableView registerNib:[UINib nibWithNibName:@"DetailsTableViewCell" bundle:nil] forCellReuseIdentifier:@"DetailsTableViewCell"];
     [self.tableView registerNib:[UINib nibWithNibName:@"ChartTableViewCell" bundle:nil] forCellReuseIdentifier:@"ChartTableViewCell"];
+    
+    LogCategory *mockLog = [LogCategory getMockLog];
+    self.logCategory =  mockLog;
+    PFQuery *query = [PFQuery queryWithClassName:@"LogCategory"];
+    [query whereKey:@"parent" equalTo:[PFUser currentUser]];
+    [query includeKey:@"entry"];
+    [query findObjectsInBackgroundWithBlock:^(NSArray *categories, NSError *error) {
+        if (categories.count > 0) {
+            LogCategory *category = [[LogCategory alloc] init];
+            category.entries = [[NSMutableArray alloc] init];
+            PFObject *parseLog = [categories objectAtIndex:0];
+            category.name = parseLog[@"name"];
+            PFQuery *entriesQuery = [PFQuery queryWithClassName:@"Entry"];
+            [entriesQuery whereKey:@"parent" equalTo:parseLog.objectId];
```

Comment:
@ChrisGuzman sometimes I see just object other places I see `.objectId` appended. does this make any difference?